monitoring/mongodb/dashboard.py

```diff
@@ -819,7 +778,7 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
     targets=[
         Target(
             expr="max by (job) (" + Metrics.MONGOS_SHARDING_CHUNKS_IS_BALANCED() + ")",
-            legendFormat="{{pod}}",
+            legendFormat="{{ job }}",
```

Comment:
there was a comment (earlier) about consistently choosing between `job` and `pod` in legends: at other places with do it `by(pod)`, should the fix here and line 833 not be do do `max by(pod)` ?

```suggestion
expr="max by (pod) (" + Metrics.MONGOS_SHARDING_CHUNKS_IS_BALANCED() + ")",
            legendFormat="{{pod}}",
```